<?xml version="1.0"?>
<launch>
    <arg name="subscript" default="" />
    <arg name="enable_logging" default="false" />
    <!-- Whether to use perfect gazebo position of PX4 LPE/EKF2 Position  -->
    <arg name="use_gazebo_ground_truth" value="true"/>
    <arg name="mapping_param_file" default="$(find semantic_occupancy_mapping_3d)/config/semantic_mapping.yaml" />
 
    <!-- Semantic pointcloud topic -->
    <arg name="pcl_topic" default="/semantic_pcl/semantic_pcl"/>
    <arg name="stl_file_path" default="" />

    <!-- Sem Mapper  -->
    <rosparam command="load" file="$(arg mapping_param_file)" />
    <node pkg="semantic_occupancy_mapping_3d" name="sem_node" type="sem_node" output="screen">
        <remap from="/mavros/local_position/pose" to="/uav1/mavros/local_position/pose"/>
        <remap from="odometry" to="/uav1/odometry/odom_main"/>
    </node>
    
</launch>
